lib/Twig/Template.php

```diff
@@ -603,6 +603,8 @@ protected function getAttribute($object, $item, array $arguments = array(), $typ
                 }
             } else {
                 $methods = get_class_methods($object);
+                // sort values to have consistent behavior with lines 611 to 631
+                sort($methods);
```

Comment:
The sorting should be outside the ``else``, as it is necessary also for ``Twig_Template`` child classes actually.